run-tests-with-build-kogito-examples-parent/kogito-examples-scm-checkout-test/pom.xml

```diff
@@ -26,7 +26,11 @@
       kogito.bom.version,
       optaplanner.bom.group-id,
       optaplanner.bom.artifact-id,
-      optaplanner.bom.version
+      optaplanner.bom.version,
```

Comment:
do we still need the optaplanner here ?
AFAIK, optaplanner is out of kogito examples